doc/smart_contract/tok_ctrt_no_split.md

```diff
@@ -0,0 +1,466 @@
+
+const tc = await tok.TokCtrtWithoutSplit.register(
+  acnt, // by
+  10000, // max
+  100, // unit
+  'token description', // tokDescription
+  'certificate description' // ctrtDescription
+);
+
+console.log('ctrtID:', tc.ctrtId.data);
+console.log('tokID: ', tc.getTokId(await tc.getLastTokIdx()).data);
+```
+
+Example output
+
+```
+ctrtID: CEuFhD7N4sRfGnx6HZnCLnnd517Zgy19p4Q
+tokID:  TWsipaf2H3TPkDXwp2usNX9fdStSJ7SPPJzAc7nZo
+```
+
+### From Existing Contract
+
+`tokCtrtId` is the ctrtID of previously registered token.
+
+```javascript
+import * as tok from './src/contract/tok_ctrt_no_split.js';
+
+// ch: Chain from './src/chain.js'
+
+const tokCtrtId = 'CEuFhD7N4sRfGnx6HZnCLnnd517Zgy19p4Q';
+const tc = new tok.TokCtrtWithoutSplit(tokCtrtId, ch);
+```
+
+### Querying
+
+#### Issuer
+
+The address that has the issuing right of the Token contract instance.
+
+```javascript
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+console.log(await tc.getIssuer());
+```
+
+Example output
+
+```
+Addr { data: 'ATracVxHwdYF394gXEawdZe9stB9yLH6V7q' }
+```
+
+#### Maker
+
+The address that made this Token contract instance.
+
+```javascript
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+console.log(await tc.getMaker());
+```
+
+Example output
+
+```
+Addr { data: 'ATracVxHwdYF394gXEawdZe9stB9yLH6V7q' }
+```
+
+#### Token ID
+
+The token ID of the token defined in the token contract instance.
+
+Note that theoretically a token contract instance can have multiple kinds of token, it is restricted to 1 kind of token per token contract instance. In other word, the token ID is of the token index `0`.
+
+```javascript
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+console.log(tokB.getTokId(0));
+```
+
+Example output
+
+```
+TokenID { data: 'TWsipaf2H3TPkDXwp2usNX9fdStSJ7SPPJzAc7nZo' }
+```
+
+#### Unit
+
+The unit of the token defined in this token contract instance.
+
+```javascript
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+console.log(await tc.getUnit());
+```
+
+Example output
+
+```
+100
+```
+
+#### Token Balance
+
+Query the balance of the token defined in the contract for the given user.
+
+```javascript
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+// acnt: Account from './src/account.js'
+
+console.log(await tc.getTokBal(acnt.addr.data));
+```
+
+Example output
+
+```
+Token { data: BigNumber { s: 1, e: 5, c: [ 500000 ] }, unit: 100 }
+```
+
+### Actions
+
+#### Supersede
+
+Transfer the issuer role of the contract to a new user.
+The maker of the contract has the privilege to take this action.
+
+```javascript
+// acnt0: Account from './src/account.js'
+// acnt1: Account from './src/account.js'
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+const resp = await tc.supersede(
+  acnt0, // by
+  acnt1.addr.data, // newIssuer
+  'giving rights to issue token to acnt1' // attachment
```

Comment:
it is not necessary to let user write attachment here. 
```suggestion

```

Reply:
we should remove the `attachment` part for the other transactions. Users may make mistake with more parameters.